src/PDFs/physics/SquareDalitzEffPdf.cu

```diff
@@ -16,7 +16,7 @@ __device__ fptype thetaprime(fptype m12, fptype m13, fptype mD, fptype mKS0, fpt
     fptype denum = sqrt(((m23 - mh1 * mh1 + mh2 * mh2) * (m23 - mh1 * mh1 + mh2 * mh2) - 4 * m23 * mh2 * mh2))
                    * sqrt(((mD * mD - mKS0 * mKS0 - m23) * (mD * mD - mKS0 * mKS0 - m23) - 4 * m23 * mKS0 * mKS0));
 

```

Comment:
Maybe try adding:

```cpp
using std::isnan;
```

Just above the first place it is used? Just make sure it works in CUDA mode.